doc/build-windows.md

```diff
@@ -129,6 +129,17 @@ Then build using:
 
 For further documentation on the depends system see [README.md](../depends/README.md) in the depends directory.
 
+Create installer
+-----------------
+
+The following steps are necessary to create the binary installer. In order to create it, the system must
+have [NSIS](https://nsis.sourceforge.io/Main_Page) installed.
```

Comment:
I think you should mention that it should be installed *before* running `./configure`, if you install it just before the `make` step in the steps below it's not going to work.